Firmware/mmu.cpp

```diff
@@ -1367,6 +1367,7 @@ bFilamentAction=false;                            // NOT in "mmu_load_to_nozzle_
 	manage_response(true, true, MMU_TCODE_MOVE);
 	mmu_continue_loading(false);
 	mmu_extruder = tmp_extruder; //filament change is finished
+	if (current_position[Z_AXIS] < 20) current_position[Z_AXIS] += 30;
```

Comment:
Please add 
```
plan_buffer_line(...
st_synchronize();
```
calls after this line to make it more clear what is going on there.

Reply:
Or even better to avoid copy pasting code, find original occurrence of this functionality (lift nozzle when loading filament), extract the code to meaningfully named function and call this function from both places.